app/views/ChangeAvatarView/index.tsx

```diff
@@ -224,40 +259,46 @@ const ChangeAvatarView = () => {
 							}
 						/>
 					) : null}
-					<Button
-						title={I18n.t('Take_a_photo')}
-						type='secondary'
-						disabled={saving}
-						backgroundColor={colors.buttonBackgroundSecondaryDefault}
-						onPress={() => pickImage(true)}
-						testID='change-avatar-view-take-a-photo'
-					/>
-					<Button
-						title={I18n.t('Upload_image')}
-						type='secondary'
-						disabled={saving}
-						backgroundColor={colors.buttonBackgroundSecondaryDefault}
-						onPress={pickImage}
-						testID='change-avatar-view-upload-image'
-					/>
-					{context === 'room' ? (
+					<View style={styles.buttons}>
 						<Button
-							title={I18n.t('Delete_image')}
-							type='primary'
+							title={I18n.t('Take_a_photo')}
+							type='secondary'
 							disabled={saving}
-							backgroundColor={colors.buttonBackgroundDangerDefault}
-							onPress={() => dispatchAvatar({ type: AvatarStateActions.RESET_ROOM_AVATAR, payload: { data: null } })}
-							testID='change-avatar-view-delete-my-account'
+							backgroundColor={colors.buttonBackgroundSecondaryDefault}
+							onPress={() => pickImage(true)}
+							testID='change-avatar-view-take-a-photo'
+							style={{ marginBottom: 0, marginTop: 0 }}
```

Comment:
Several duplicated inline styles.
You should create a style for them.

Reply:
adjusted.